util/regexp-assemble/data/932105.data

```diff
@@ -91,9 +91,10 @@ $\(\(
 (?:[\?\*\[\]\(\)\-\|+\w'\"\./\x5c]+/)?[\x5c'\"]*
 ##!=>
 
-  ##!> cmdline unix
+##!> cmdline unix
```

Comment:
Should be indented, it's nested within `assemble`.

Reply:
Noted.